<launch>
  <node
    pkg="mrobosub_perception"
    type="webcam_pub.py"
    name="webcam_pub"
    output="screen"
  />
  <node
    pkg="mrobosub_perception"
    type="hsv_tuner.py"
    name="hsv_tuner"
    output="screen"
  >
    <rosparam command="load" file="$(find mrobosub_perception)/params/pathmarker.yaml" />
  </node>
</launch>
